nCompiler/R/NF_CompilerClass.R

```diff
@@ -142,16 +142,33 @@ processNFstages <- function(NFcompiler,
     )
     debug <- controlFull$debug
     debugCpp <- FALSE##controlFull$debugCpp
+    logging <- controlFull$logging
+    ## to make logging option accessible in handlers, must set .nOptions directly
+    if (logging) .nOptions$compilerOptions[['logging']]  <- TRUE
```

Comment:
We have a generic `set_nOption`, but that's awkward here because it is a nested option.  We may want to make a clean interface to get and set compiler options.

Reply:
Agreed. I wanted to use a generic method here but ran into the nesting issue. How do you feel about the pattern I use in `updateOperatorDef` and `getOperatorDef` in [compile_aaa_operatorLists.R](https://github.com/nimble-dev/nCompiler/blob/c92ab9755d44d4a6422d558483819b7755e28d2b/nCompiler/R/compile_aaa_operatorLists.R#L20-L29)?